### DISCUSSION TITLE
How can I graciously tell that `find` didn't find searched element in order to continue test execution?

### DISCUSSION THREAD

**Discussion by quad5** [cypress-io/cypress#14245] [Questions and Help]
If `find` didn't find search element, what can we do to continue the execution? Currently, it just stops test and continue to run when it found it.

`cy.get('body').find('.closeX', {timeout: 20000}).then((el) => {})`

Error message: `Timed out retrying: Expected to find element: .closeX, but never found it. Queried from element: <body>`

**Comment by bahmutov** [comment]
It is an example of conditional testing https://on.cypress.io/conditional-testing which we do not recommend. Maybe in your case

```js
cy.wait(20000)
cy.get('body').then(body => {
  // now use body.find
  // and if the button is there, click on it
  // using cy.wrap(found).click()
})
```
